### PR TITLE
[W31] evanxd: 08/01 - 08/05

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

File: evanxd.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
-## 07/25 ~ 07/29 ##
+## 08/01 ~ 08/05 ##
 
 * [Firefox]
   - Dev Tools
-    - Bug 1286892 - Inspector sidebar tabs now have a bigger font-size, r+ and landed
-    - Bug 1282791 - Reps: uninteresting props algorithm is wrong, landed
+    - Bug 1289912 - Can't scroll in JSON Viewer in Nightly, landed
```
Comment (timdream) [inline evanxd.md]: Could you help me by document the status on a new line? This will make diff more readable.

File: evanxd.md [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
     - Bug 1284838 - Reps: render events more uniformly, review?
-    - Bug 1289912 - Can't scroll in JSON Viewer in Nightly, review?
+      - Updated patch for the review comments, still on review process.
+    - Bug 1288854 - Pressing "escape" to cancel the eye dropper does not always cancel it
+      - Duplicate of Bug 1289433.
+    - Bug 1289062 - Order the Event's properties
+      - Discussed the interesting Event rep's properties with reviewer(Honza).
+  - Location Bar
+    - Bug 1256074 - Always present a 'search' option, even location bar's contents are detected as URLs
+      - Traced code to prepare write patch for the bug. Will write patch this week.
```
Comment (timdream) [inline evanxd.md]: 👍

File: evanxd.md (hunk 1 above)
Comment (evanxd) [inline evanxd.md]: Sure, let's do something like below in the future.

- Bug 1289912 - Can't scroll in JSON Viewer in Nightly
  - landed

Comment (timdream) [inline evanxd.md]: Yes, thank you!

**Pull request merged**